dash/dash-renderer/src/libraries/LibraryComponent.tsx

```diff
@@ -15,16 +14,6 @@ type LibraryComponentProps = {
 
 const LibraryComponent = (props: LibraryComponentProps) => {
     const {props_check, namespace, type, ...rest} = props;
-
-    const context = useContext(LibrariesContext);
```

Comment:
So I guess this was left over from a previous iteration but not needed the way you finally implemented the feature?